exercises/defining_models/scaffold/models/book.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
+
+const orm = require('ormnomnom')
+const Author = require('./author.js')
```

Comment:
I get the feeling `fluent-json-schema` will help here (I mistakenly recommended `fluent-api-schema` earlier, which does not exist!)